Sources/SwiftLibraryPluginProvider/LibraryPluginProvider.swift

```diff
@@ -0,0 +1,204 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6.0)
+public import SwiftSyntaxMacros
+@_spi(PluginMessage) public import SwiftCompilerPluginMessageHandling
+// NOTE: Do not use '_'
```

Comment:
```suggestion
// NOTE: Do not use '_SwiftSyntaxCShims' for 'dlopen' and 'LoadLibraryW' (Windows) because we don't want other modules depend on 'WinSDK'.
```